syntax = "proto3";

message Transaction {
  string transaction_id = 1;
  string account_number = 2;
  string transaction_reference = 3;
  string transaction_datetime = 4;
  double amount = 5;
}

